src/hooks/productData/error_tracking.tsx

```diff
@@ -91,14 +105,39 @@ export const errorTracking = {
             ],
         },
         {
-            title: 'Stack traces',
-            headline: 'Stack traces',
-            description:
-                "Get code context automatically with PostHog's server-side libraries, or upload source maps for front-end frameworks",
+            title: 'Assign and triage',
+            headline: 'Assign and triage',
+            description: 'Auto-assign issues to individuals or groups',
             images: [
                 {
-                    src: 'https://res.cloudinary.com/dmukukwp6/image/upload/screenshot_trace_3fc569059c.png',
-                    alt: 'Stack traces',
+                    src: 'https://res.cloudinary.com/dmukukwp6/image/upload/screenshot_assign_4c9bb9ee60.png',
+                    alt: 'Assign and triage',
+                    stylize: true,
+                    shadow: true,
+                },
+            ],
+        },
+        {
+            title: 'Investigate and resolve',
+            headline: 'Investigate and resolve',
+            description: 'Use PostHog session replay to investigate and resolve issues with complete customer context',
+            images: [
+                {
+                    src: 'https://res.cloudinary.com/dmukukwp6/image/upload/error_tracking_session_replay_investigate_da4ee40642.gif',
+                    alt: 'Investigate and resolve',
+                    stylize: true,
+                    shadow: true,
+                },
+            ],
+        },
+        {
+            title: 'Target affected users',
+            headline: 'Target affected users',
+            description: 'Roll out changes with PostHog feature flags to target only affected users',
```

Comment:
Not really sure what we're suggesting this feature does. Conscious we don't yet have a full rollback integration so don't want to overly imply so
```suggestion
            description: 'Revert feature flag roll out to users who are affected by an issue',
```